repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,6 +4,6 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "cd hdfs-mesos* && ./bin/hdfs-mesos",
+  "cmd": "{{#hdfs/custom-config}}echo {{hdfs/custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs/custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
```

Comment:
How are we planning to document that the user can generate this file by using the hdfs cli? Should we use the package's description to explain this? https://github.com/mesosphere/universe/blob/master/repo/meta/schema/package-schema.json#L20